deploy/imagemanifestvuln.crd.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
New version of `CustomResourceDefinition`. It will still mirror to `v1beta1`, but you can only create `v1` in Kubernetes 1.16+.